docs/subvolume.md

```diff
@@ -40,16 +40,20 @@ kubectl rook-ceph subvolume ls --stale
 ## delete
 
 ```bash
-kubectl rook-ceph subvolume delete csi-vol-427774b4-340b-11ed-8d66-0242ac110004 ocs-storagecluster csi
+kubectl rook-ceph subvolume delete ocs-storagecluster-cephfilesystem csi-vol-427774b4-340b-11ed-8d66-0242ac110005 csi
 
 # Info: subvolume csi-vol-427774b4-340b-11ed-8d66-0242ac110004 deleted
+# Info: omap object:"csi.volume.427774b4-340b-11ed-8d66-0242ac110005" deleted
+# Info: omap key:"csi.volume.pvc-fca205e5-8788-4132-979c-e210c0133182" deleted
 
 ```
 
 ```bash
-kubectl rook-ceph subvolume delete csi-vol-427774b4-340b-11ed-8d66-0242ac110004,csi-vol-427774b4-340b-11ed-8d66-0242ac110005 ocs-storagecluster csi
+kubectl rook-ceph subvolume delete ocs-storagecluster-cephfilesystem csi-vol-test csi
 
-# Info: subvolume csi-vol-427774b4-340b-11ed-8d66-0242ac110004 deleted
-# Info: subvolume csi-vol-427774b4-340b-11ed-8d66-0242ac110004 deleted
+# Info: subvolume "csi-vol-test" deleted
+# Error: error getting omap keys ocs-storagecluster-cephfilesystem-metadata/test.subvolume:
+# (2) No such file or directory
+# . failed to run command. command terminated with exit code 1%!(EXTRA string=failed to list omapval)
```

Comment:
Incase the subvolume is created from backend and it's stale, there will be no omap created for it.  In that case if we try to delete the omap, it will through this error.

Reply:
let's document the happy case output.